src/add-remove.js

```diff
@@ -0,0 +1,100 @@
+import iconDots from './icon_dots.png';
+import iconDelete from './delete.png';
+import './style.css';
+
+const addList = document.body.querySelector('.add-list');
+const inputList = document.body.querySelector('.input-list');
+const uList = document.body.querySelector('.ulist-child');
+export const todoArray = [];
+
+addList.addEventListener('click', () => {
+  if (inputList.value !== '') {
+    const des = inputList.value;
+    todoArray.push(
+      { description: des, completed: false, index: 0 },
+    );
+  }
+  inputList.value = '';
+  /* eslint-disable */
+  createList();
```

Comment:
- [ ] Kindly try to remove the disabled linters line and fix it manually as asked by the previous reviewer.